js/MVP/containers/__tests__/DialogmoteContainer.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
+import DialogmoteContainer from '../DialogmoteContainer';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+
+describe('DialogmoteContainer', () => {
+  test('should render', () => {
+    const { container } = render(
+      <DialogmoteContainer title="Test title" breadcrumb={[]}>
+        <div>Test div</div>
+      </DialogmoteContainer>
+    );
+
+    expect(container).toHaveTextContent('Test div');
+  });
+
+  test('should render tilbakeknapp and invoke goBack on click', () => {
+    const goBackMock = jest.fn();
+    const historyStub = createMemoryHistory();
+    const history = { ...historyStub, goBack: goBackMock };
+
+    render(
+      <Router history={history}>
+        <DialogmoteContainer title="Test title" breadcrumb={[]} displayTilbakeknapp={true}>
+          <div>Test div</div>
+        </DialogmoteContainer>
+      </Router>
+    );
+
+    const tilbakeknapp = screen.getByRole('button', { name: 'Tilbake' });
```

Comment:
Jeg mener å huske at vi snakket litt om å bruke en slags test-id for å finne komponenter. Ser at det er noe som heter `screen.getByTestId()`. Er det fornuftig å bruke det? Vi bør i tilfelle ha en felles enighet om når man skal bruke hva.

Reply:
Jeg har fulgt denne så langt det lar seg gjøre: https://testing-library.com/docs/queries/about/#priority

Vi kan lage guidelines for det når vi begynner med Cypress, godt innspill :)